andela/ah-frontend-stark
# #160583965 Users should be able to comment on articles

#### What does this PR do?
Users should be able to comment on articles
#### Description of Task to be completed?
Comments allow authors to get feedback on their articles.
We all love to know what people think about what we put out to the world
#### How should this be manually tested?
- clone the project
```console
git clone https://github.com/andela/ah-frontend-stark.git
```
- move into the project folder
```console 
cd ah-frontend-stark
```
- install the dependency
``` console 
yarn install
```
- run the project
```console
yarn start
```
navigate to one of the comment and there should be a comment box
```
url /article/some-other-stuff-to-talk-about12345
```
#### What are the relevant pivotal tracker stories?
[#160583965](https://www.pivotaltracker.com/n/projects/2198584/stories/160583965)
#### Screenshots (if appropriate)
<img width="1434" alt="screen shot 2018-11-12 at 15 19 46" src="https://user-images.githubusercontent.com/26383328/48347165-853e5780-e68e-11e8-9d22-bfb81dfc9822.png">
